drivers/service_upgrade.go

```diff
@@ -263,7 +264,7 @@ func wait(apiClient *client.RancherClient, service *client.Service) error {
 func IsValidTag(tag string) error {
 	match := regTag.FindAllString(tag, -1)
 	if len(match) == 0 || len(match[0]) > 128 || (len(match[0]) != len(tag)) {
-		return fmt.Errorf("Invalid tag %s, tag length must be < 128, must contain [a-zA-Z0-9.-_] characters only, cannot start with '.' or '-'", tag)
+		return fmt.Errorf("Invalid tag %s, tag length must be < 128, must contain [a-zA-Z0-9.-_*] characters only, cannot start with '.' or '-'", tag)
```

Comment:
@mrajashree So which character do you suggest as wildcard? If we choose one that is valid in docker tags, we would have no way of knowing if the user wanted wildcard matching or exactly this tag. (Also * is wildly used as wildcard)